src/pyhf/optimize/opt_minuit.py

```diff
@@ -118,9 +127,16 @@ def _minimize(
         if return_uncertainties:
             unc = minimizer.np_errors()
 
+        # cov = None
+        corr = None
+        if return_correlations:
+            # cov = minimizer.hesse(maxcall=maxiter)
```

Comment:
In regards to

> Do we want to support the covariance too?

I guess sure? I haven't thought about this much yet, so is there a clear reason why we might not want to? It seems that since we're already adding correlations we might as well add this in as well as I suspect that people will want it.

Reply:
Users can get it from the fit object. It's not as commonly requested as the correlation (normalized covariance).